book/src/specs/kimchi.md

```diff
@@ -1450,10 +1536,17 @@ pub struct VerifierIndex<G: KimchiCurve> {
     #[serde(bound = "PolyComm<G>: Serialize + DeserializeOwned")]
     pub chacha_comm: Option<[PolyComm<G>; 4]>,
 
-    // Range check gates polynomial commitments
     #[serde(bound = "PolyComm<G>: Serialize + DeserializeOwned")]
     pub range_check_comm: Option<[PolyComm<G>; range_check::gadget::GATE_COUNT]>,
 
+    // Foreign field modulus
+    //#[serde(bound = "Option<ForeignElement<G::ScalarField, LIMB_COUNT>>: Serialize + DeserializeOwned")]
+    pub foreign_field_modulus: Option<BigUint>,
```

Comment:
Replace comment with `#[serde(bound = "Option<BigUint>: Serialize + DeserializeOwned")]?`

Reply:
Oops thanks

Reply:
actually not necessary because of the serde feature of bigint